### PR TITLE
Add molecules section

### PR DESCRIPTION
**Pull request by ayaosama05**
-- add molecules section

### CODE REVIEW

File: static/public/index.html [hunk 1]
```diff
@@ -9056,6 +9056,768 @@ <h1 class="text-hero-large-bold mb-4">Styles :</h1>
...
+       </div>
+     </div>
+     <div class="col-span-3 text-hero-large-bold">navigation</div>
+     <div class="col-span-3">
+       <div class="logo">
+         <p>pickly</p>
+       </div>
+       <div class="nav">
+         <ul>
+           <li><a href="#">
```
Comment (AymanMorsy) [inline static/public/index.html:14282]: needs to update background-color of SVG container

File: static/public/index.html [hunk 2]
```diff
@@ -9056,6 +9056,768 @@ <h1 class="text-hero-large-bold mb-4">Styles :</h1>
...
+             </span>
+             <div class="mt-xxs flex border border-accent py-2.5 px-4 rounded-md w-full">
+               <input type="text" class=" bg-transparent text-sm w-full text-accent focus:outline-none" value="add option">
+             </div>
+           </div>
+         </div>
+       </div>
+     </div>
+     <div class="col-span-3 text-hero-large-bold">image caption & user info</div>
+     <div class="col-span flex flex-row text-dark items-center justify-start gap-xs">
```
Comment (AymanMorsy) [inline static/public/index.html:14096]: add border-radius on the caption card

**Review by AymanMorsy — CHANGES_REQUESTED** [review]

**Comment by hebasaadosman** [comment]
good job aya 👍 
very small notes
1- background of image upload not white 
2- 25% dislike need to justify on image

**Review by hebasaadosman — DISMISSED** [review]
LGTM

**Review by hebasaadosman — DISMISSED** [review]

**Review by hebasaadosman — APPROVED** [review]

**Pull request merged**